components/blocks/index.tsx

```
@@ -0,0 +1,197 @@
+    style,
+    labelStyle,
+    className,
+  }: GroupBlockType) => (
+    <div className="space-y-2" data-path={dataPath}>
+      {label && dataPath !== "abilities" && (
+        <Label style={labelStyle} className="font-bold text-center block">
+          {label}
+        </Label>
+      )}
+      <GroupContent
+        children={children}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)